json/products/mozilla-concrete-brick.json

```diff
@@ -1,8 +1,11 @@
 {
   "description": {
-    "en": "Such a fine brick"
+    "en": "Brick"
+  }, 
+  "img": "https://raw.githubusercontent.com/mozilla/payments-config/master/payments_config/assets/default.png", 
+  "price": {
+    "en": "$10.00"
   }, 
-  "img": "https://raw.githubusercontent.com/mozilla/payments-config/master/payments_config/assets/brick.png", 
```

Comment:
actually, now I'm unsure of this. Did you mean to set the brick image to default? It looks like you meant to keep the brick image but declare the default as a fallback in ``__init__``. So maybe this should not have been deleted?